### PR TITLE
Recaptcha

### PR DESCRIPTION
**Pull request by rawOrlando**
Adds a recaptcha checkbox at the end of the contact form page. merge https://gitlab.it.tufts.edu/ets/ansible/ets-ansible-roles/-/merge_requests/12 first. so that keys exist on the host. 

recaptcha info: https://www.google.com/u/1/recaptcha/admin

todo:
- [x] Error message shows when submit without the check box
- [x] Move the keys to a more secret place, likely ansible

### CODE REVIEW

**Comment by rawOrlando** [comment]
I copied and app/controllers/hyrax/contact_form_controller.rb from hyrax to modify it slightly, should I also copy the spec for coveralls?

File: app/controllers/hyrax/contact_form_controller.rb [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
```
Comment (mkorcy) [inline app/controllers/hyrax/contact_form_controller.rb:1]: for upgrade purposes can you add a note about why you copied this in to the source tree.

**Review by mkorcy — CHANGES_REQUESTED** [review]
2 small changes.  one for clarity on copying the file from hyrax locally just so i can keep track as we move towards hyrax 3/4 and just a request to delete a file we're not using anymore.

File: .travis.yml
[the remark this answers is not in the document]
Comment (rawOrlando) [reply .travis.yml:1]: What do you mean?

**Review by mkorcy — APPROVED** [review]
👍

**Pull request merged**